external/fv3fit/tests/test_xr_prediction.py

```diff
@@ -39,3 +42,81 @@ def _get_dummy_model(input_shape):
     out_ = tf.keras.layers.Lambda(lambda x: x)(in_)
     model = tf.keras.Model(inputs=in_, outputs=out_)
     return model
+
+
+@io.register("constant-array")
```

Comment:
Nit: We've [sometimes](https://github.com/ai2cm/fv3net/blob/eb8fc6edcf26ed4a09d74ff6edd961191813060f/external/fv3fit/tests/test_register_model.py#L17) used context managers to avoid adding test cases to registries, i.e., add it only for the time the test is run.